src/validate.ts

```diff
@@ -30,14 +30,15 @@ function responseValidation(apiResponse: object, schema: object) {
  * @param responseType User defined intent response
  * @return Errors from AJV validation, if any. Undefined otherwise.
  */
-export function validate(apiResponse: object, responseType: string) {
-  if (responseType === 'sync') {
+export function validate(req: object, apiResponse: object) {
```

Comment:
Nit: I'd suggest a more descriptive param name for this.

Reply:
for the request param?

Reply:
You could make it slightly more explicit.

Reply:
Yes, sorry. For the request.